### PR TITLE
Exposing the 'stop_historu_eou_th' parameter

### PR DESCRIPTION
**Pull request by sarane22**
With these changes, clients can now configure `stop_eou_threshold` for streaming ASR:
The new parameters introduced are:
`--stop_eou_threshold = float`

### CODE REVIEW

**Comment by sarane22** [comment]
TODO: Update the `nvidia-riva/common` SHA once changes are merged.

File: riva/clients/asr/riva_streaming_asr_client.cc
Comment (rmittal-github) [inline riva/clients/asr/riva_streaming_asr_client.cc]: we can use built-in VLOG here to avoid adding the new flag

File: riva/clients/asr/riva_asr_client.cc
Comment (rmittal-github) [inline riva/clients/asr/riva_asr_client.cc]: need to add stop_threshold_eou_ here, right?

File: riva/clients/asr/riva_asr_client.cc
Comment (rmittal-github) [inline riva/clients/asr/riva_asr_client.cc:291]: and here as well

File: riva/clients/asr/riva_asr_client.cc
Comment (sarane22) [reply riva/clients/asr/riva_asr_client.cc]: Updated with above commit.

File: riva/clients/asr/riva_asr_client.cc
Comment (sarane22) [reply riva/clients/asr/riva_asr_client.cc:291]: Updated with above commit.

File: WORKSPACE
Comment (rmittal-github) [inline WORKSPACE]: reminder comment, update this once proto change gets merged

**Comment by rmittal-github** [comment]
LGTM, will merge once proto change is merged and server pipeline is ok

**Review by rmittal-github — APPROVED** [review]

**Pull request merged**
